packages/theme-toolkit/src/component-stories-candidate.tsx

```diff
@@ -0,0 +1,377 @@
+        'nl.code.background-color',
+        'nl.code.font-size',
+        'nl.code.font-family',
+        'nl.code.line-height',
+      ],
+    },
+  },
+  {
+    storyId: 'react-nl-code-block--default',
+    name: 'Candidate Code Block',
+    render: () => (
+      <CodeBlock>
+        {`<!DOCTYPE html>
+        <html lang="nl" dir="ltr">
+          <head>
+            <title>NL Design System</title>
+            <meta charset="utf-8"/>
+          </head>
+          <body>
+            <h1>NL Design System</h1>
+          </body>
+        </html>`}
+      </CodeBlock>
+    ),
+    detectTokens: {
+      anyOf: [
+        'nl.code-block.font-family',
+        'nl.code-block.background-color',
+        'nl.code-block.color',
+        'nl.code-block.font-size',
+        'nl.code-block.line-height',
+        'nl.code-block.margin-block-start',
+        'nl.code-block.margin-block-end',
+        'nl.code-block.margin-inline-start',
+        'nl.code-block.margin-inline-end',
+        'nl.code-block.padding-block-start',
+        'nl.code-block.padding-block-end',
+        'nl.code-block.padding-inline-start',
+        'nl.code-block.padding-inline-end',
+      ],
+    },
+  },
+  {
+    storyId: 'react-nl-color-sample',
+    name: 'Candidate Color Sample',
+    render: () => <ColorSample value="#007DAD" />,
+    detectTokens: {
+      anyOf: [
+        'nl.color-sample.background-color',
+        'nl.color-sample.border-color',
+        'nl.color-sample.border-width',
+        'nl.color-sample.border-radius',
+        'nl.color-sample.block-size',
+        'nl.color-sample.inline-size',
+      ],
+    },
+  },
+  {
+    storyId: 'react-nl-color-sample-dark',
+    name: 'Candidate Color Sample Dark',
+    render: () => <ColorSample value="#007DAD" />,
+    detectTokens: {
+      anyOf: ['nl.color-sample.dark.border-color'],
+    },
+  },
+  {
+    storyId: 'react-nl-color-sample-light',
+    name: 'Candidate Color Sample Light',
+    render: () => <ColorSample value="#007DAD" />,
+    detectTokens: {
+      anyOf: ['nl.color-sample.light.border-color'],
+    },
+  },
+  {
+    storyId: 'react-nl-data-badge--default',
+    group: STORY_GROUPS['BADGES'],
+    name: 'Candidate Data Badge',
+    render: () => <DataBadge>Category 1</DataBadge>,
+    detectTokens: {
+      anyOf: [
+        'nl.data-badge.color',
+        'nl.data-badge.border-radius',
+        'nl.data-badge.border-width',
+        'nl.data-badge.background-color',
+        'nl.data-badge.font-size',
+        'nl.data-badge.font-weight',
+        'nl.data-badge.line-height',
+        'nl.data-badge.min-block-size',
+        'nl.data-badge.min-inline-size',
+        'nl.data-badge.padding-block',
+        'nl.data-badge.padding-inline',
+      ],
+    },
+  },
+  {
+    storyId: 'react-nl-heading--level-1',
+    component: 'nl-heading',
+    group: STORY_GROUPS['HEADINGS'],
+    name: 'Candidate Heading - Level 1',
+    render: () => (
+      <Heading role="presentation" level={1}>
```

Comment:
Ik zou niet standaard `role="presentation"` gebruiken. Als AXE anders klaagt, moeten we denk ik ergens AXE ignore toevoegen.